### PR TITLE
Corrected copyright strings Take 3

### PR DESCRIPTION
**Pull request by islemaster**
Reverts code-dot-org/code-dot-org#6082

This should not be this hard guys.

### CODE REVIEW

**Comment by islemaster** [comment]
The eyes test this broke was on the text compression widget, which has a hard-coded value for its fixed layout that relates to the footer height.

I went ahead and converted text compression widget to use small footer, because it was the only fixed-viewport page using the large footer, and the new larger copyright text eats up tons of screen space in the large footer.

Here's a 1000x700 before image (production, original copyright text):
screenshot from 2015-12-15 12-53-48

Here's now, with the small footer:
screenshot from 2015-12-15 12-54-54

File: dashboard/app/models/game.rb [hunk 1]
```diff
@@ -127,7 +127,7 @@ def uses_pusher?
   end
 
   def uses_small_footer?
-    app == NETSIM || app == APPLAB
+    app == NETSIM || app == APPLAB || name == "TextCompression"
```
Comment (joshlory) [inline dashboard/app/models/game.rb]: If you add a `TEXT_COMPRESSION = 'text_compression'` constant you can follow the existing pattern here.

**Comment by joshlory** [comment]
LGTM

File: dashboard/app/models/game.rb (hunk 1 above)
Comment (islemaster) [inline dashboard/app/models/game.rb]: :+1: Sounds good.  The convention looks like it's only been applied to the `app` up until now - comparisons against `name` are always against string literals in this file.  No reason we can't generalize that though.

File: dashboard/app/models/game.rb [hunk 2]
```diff
@@ -45,6 +45,7 @@ def self.custom_maze
   MAZE = 'maze'
   CALC = 'calc'
   EVAL = 'eval'
+  TEXT_COMPRESSION_NAME = 'TextCompression';
```
Comment (joshlory) [inline dashboard/app/models/game.rb]:
``` ruby
TEXT_COMPRESSION = 'text_compression'
```

File: dashboard/app/models/game.rb [hunk 3]
```diff
@@ -83,7 +84,7 @@ def self.pixelation
   end
 
   def self.text_compression
-    @@game_text_compression ||= find_by_name("TextCompression")
+    @@game_text_compression ||= find_by_name(TEXT_COMPRESSION_NAME)
```
Comment (joshlory) [inline dashboard/app/models/game.rb]: Undo change.

File: dashboard/app/models/game.rb [hunk 4]
```diff
@@ -127,7 +128,7 @@ def uses_pusher?
   end
 
   def uses_small_footer?
-    app == NETSIM || app == APPLAB
+    app == NETSIM || app == APPLAB || name == TEXT_COMPRESSION_NAME
```
Comment (joshlory) [inline dashboard/app/models/game.rb]:
``` ruby
app == NETSIM || app == APPLAB || app == TEXT_COMPRESSION
```

File: dashboard/app/models/game.rb (hunk 1 above)
Comment (joshlory) [inline dashboard/app/models/game.rb]:
``` bash
irb(main):001:0> TextCompression.first.game.app
  TextCompression Load (2.7ms)  SELECT  `levels`.* FROM `levels` WHERE `levels`.`type` IN ('TextCompression')  ORDER BY `levels`.`id` ASC LIMIT 1
  Game Load (0.3ms)  SELECT  `games`.* FROM `games` WHERE `games`.`id` = 45 LIMIT 1
=> "text_compression"
```

**Comment by islemaster** [comment]
Okay, I think this is good to go.  Holding off until we have a clean test run though, to avoid confusion.

**Pull request merged**